packages/bpk-component-modal/src/BpkModal.js

```diff
@@ -53,8 +54,9 @@ const BpkModal = props => {
       renderTarget={renderTarget}
     >
       <ScrimBpkModalDialog
-        onClose={onClose}
+        onClose={closeOnScrimClick ? onClose : () => {}}
```

Comment:
The current implementation of triggering `onClose` when the scrim is clicked is in the `BpkScrim` component no? That means the `closeOnScrimClick` prop should be acted upon in that component as opposed to here.

Reply:
Done.